bigbluebutton-html5/imports/ui/components/loader/component.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
```

Comment:
I saw this one but when removed browser says `React is not defined`, even if I am not using `React` in this component.